src/pages/Home.scss

```diff
@@ -0,0 +1,27 @@
+.back {
+    position: relative;
+    text-align: center;
+}
+
+#acm {
+    width: 100%
```

Comment:
height: 100vh instead because small screens that don't have the right aspect ratio will have large white space below